libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -130,6 +130,7 @@ string IRGenerator::generate(
 	};
```

Comment:
Can you modify the warning banner above so that it does not create a doxygen comment?